reinforcement_learning/testconfig.yml

```diff
@@ -0,0 +1,11 @@
+train_instance_count:
+    - 1
+    - 2
```

Comment:
Are we using a global config file here? I thought you mentioned multi instance will only be run for ray notebook.

Reply:
Yes, we have a single global config only. Exttra parameters passed to, say train_instance_count to a Coach notebook, will be ignored as that param is not used (or defined as a variable) in the notebook.